fuzzers/056-pip-rem/generate.tcl

```diff
@@ -1,5 +1,9 @@
 source "$::env(XRAY_DIR)/utils/utils.tcl"
 
+proc waive_drc_errors {} {
```

Comment:
This needs a comment about why it is enabled.

Reply:
Created a separate PR #787 and applied your comment there.